feature/karmalist/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "karmalistexecutor.go",
```

Comment:
I'm going to leave this as is. I actually had them as the vanilla names and switched it because it took a half step of thinking out of debugging to have a more descriptive file name.